compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -227,9 +255,13 @@ internal class ComposeContainer(
         // Re-checking the actual size if it wasn't available during init.
         onChangeWindowSize()
         onChangeWindowPosition()
+
+        lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_START)
```

Comment:
We set `started` state on 2 different states:
- when it is iconified (minimized)
- when it is added to window hierarchy.

So, we have 2 sources of truth and races between them.

To avoid that, we should either choose one source of truth, or combine `isIconified` and `isAddedToHirarchy` states.